pkgs/development/python-modules/python-on-whales/default.nix

```diff
@@ -24,6 +24,12 @@ buildPythonPackage rec {
     hash = "sha256-JjzBFVgPNnU0q5hL+RZJMs3WxbeZbBKyvsV6clUFjpE=";
   };
 
+  # pythonRuntimeDepsCheck evaluates the more complex expression to 1.10.16 (!) and fails
+  configurePhase = ''
+    substituteInPlace requirements.txt \
+      --replace "pydantic>=2,<3,!=2.0.*" "pydantic>=2.10.0,<3"
```

Comment:
Note `--replace-fail` is preferred to `--replace`

Reply:
Done. Good to know.